src/cf-buttons/src/atoms/buttons-with-icons.less

```diff
@@ -14,9 +21,9 @@
     padding-left: unit( 11px / @btn-font-size, em );
     border-left: 1px solid mix( @btn-bg, @btn-text, 50% );
     margin-left: unit( 7px / @btn-font-size, em );
+    width: 18px;
```

Comment:
em value?

Reply:
Oh shoot, that shouldn't be there, left it in from testing.